mac/Keyman4MacIM/Keyman4MacIM/KMAboutWindow/km.lproj/KMAboutWindowController.strings

```diff
@@ -0,0 +1,15 @@
+
+/* Class = "NSWindow"; title = "KeymanMac"; ObjectID = "F0z-JX-Cv5"; */
+"F0z-JX-Cv5.title" = "Keyman for macOS";
```

Comment:
Just checking if we do want the Keyman title to be localizable. I wonder if other platforms mark "Keyman" as a non-translatable string. If that's the case, the string should be hidden from crowdin translators.

I also note the English string uses an old title "KeymanMac".

Reply:
You are correct. That string does not affect the UI, so I'll delete the localization string for it.